lib/cloud.proxy.js

```diff
@@ -84,28 +85,29 @@ function IoTKitCloud(logger, deviceId, customProxy) {
             } else {
                 me.logger.info("No credentials found for MQTT");
                 me.spBProxy = undefined;
-            }       
+            }
             if (me.spBProxy != undefined ) { 
                 /** 
                 *  Sending Birth message for SparkplugB (of node/agent and for device),
                 * As per standard Birth message is mandatory to send on start before sending Data
                 */
-                me.spBProxy.nodeBirth(me.devProf, function(err) {                  
+                me.spBProxy.nodeBirth(me.devProf, function(err) {               
```

Comment:
Lots of whitespace after the function opening brace which needs removing (appreciate this is historical but even so...)